src/scripts/main.js

```diff
@@ -1,5 +1,38 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+wall.addEventListener('click', (e) => {
+  const wallCoords = wall.getBoundingClientRect();
+  const spiderCoords = spider.getBoundingClientRect();
+  const min = 0;
+  const maxX = 350;
+  const maxY = 350;
+
+  const spiderXBorder
+    = e.clientX - wallCoords.x - (spiderCoords.width / 2)
+      - (wall.offsetWidth - wall.clientWidth) / 2;
```

Comment:
Move `(wall.offsetWidth - wall.clientWidth) / 2` to a variable

Reply:
@Anastasiia-Svintsova done